pkg/webui/components/footer/index.js

```diff
@@ -31,6 +33,10 @@ const m = defineMessages({
   footer: "You are the network. Let's build this thing together.",
 })
 
+const isMinimized =
```

Comment:
IMO, we should make the `<Link />` component responsive by itself (say via `minimize` prop or something like this).
Another option would be render different version of `<Link.Anchor />` based on screen size via css media queries.

Reply:
I agree. We usually don't do responsive styling programmatically, unless forced to. You can use a combination of attached classnames and rendering different elements and show/hide them based on screen size (using css media queries).